globals.h

```diff
@@ -46,6 +46,8 @@ extern const char* g_so_pin;
 extern const char* g_reset_user_pin;
 // Security Officer PIN after token reset.  Only used if (g_token_flags & CKF_LOGIN_REQUIRED).
 extern const char* g_reset_so_pin;
+// The wrap mechanism by which the test will be carried out.
```

Comment:
nit: it would be nice to be more specific about which tests this affects.  Maybe: 
```c++
// The name of the cipher for the wrapped key in wrapping tests.
```